### PR TITLE
clean up: node dropDisabledFields

### PR DESCRIPTION
**Pull request by khenidak**
**What type of PR is this?**
/kind api-change
/kind bug
/kind cleanup
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
clean up for node drop disabled fields, a follow up for https://github.com/kubernetes/kubernetes/pull/73977 

**Special notes for your reviewer**:
none.

**Does this PR introduce a user-facing change?**:
NONE

```release-note
NONE
```

### CODE REVIEW

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

File: pkg/registry/core/node/strategy_test.go [hunk 1]
```diff
@@ -64,94 +64,163 @@ func TestSelectableFieldLabelConversions(t *testing.T) {
 	)
 }
 
-// helper creates a NodeNode with a set of PodCIDRs
-func makeNodeWithCIDRs(podCIDRs []string) *api.Node {
-	return &api.Node{
+// helper creates a NodeNode with a set of PodCIDRs, Spec.ConfigSource, Status.Config
+func makeNode(podCIDRs []string, addSpecDynamicConfig bool, addStatusDynamicConfig bool) *api.Node {
```
Comment (mlvzk) [inline pkg/registry/core/node/strategy_test.go]: briefer:
```suggestion
func makeNode(podCIDRs []string, addSpecDynamicConfig, addStatusDynamicConfig bool) *api.Node {
```

**Pull request merged**